deploifai/cloud_profile/create.py

```diff
@@ -21,39 +20,11 @@ def create(context: DeploifaiContextObj, name: str, workspace: str):
     """
     deploifai_api = context.api
 
-    user_data = deploifai_api.get_user()
-    personal_workspace, team_workspaces = parse_user_profiles(user_data)
-
-    workspaces_from_api = [personal_workspace] + team_workspaces
+    if not "username" in context.global_config["WORKSPACE"]:
+        click.secho("Error in getting workspace")
+        raise click.Abort()
```

Comment:
The error statement should be more precise
The fact that we could not get workspace means that the user has not done login through cli right?
So maybe something like please login would be a better error statement

Reply:
uh aren't you dealing with this issue now? but yeah I agree, I was just trying to finish up everything. But on top of that, yeah I think it makes sense to ask the user to login, but maybe also make sure they choose the right workspace to create a cloud profile in

Reply:
I just wrote that so i could reference it later